### PR TITLE
Split images based on scene descriptions

### PR DESCRIPTION
**Pull request by jodreen**

### CODE REVIEW

File: code/utils/tests/test_scene_slicer.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    assert ss.scene_desc is not None
+    assert ss.scene_keys is not None
+    assert ss.segment_duration == [902, 882, 876, 976, 924, 878, 1086, 673.4]
+
+
+def test_scene_slicer_dict():
+    ss = scene_slicer.SceneSlicer(1, "")
+    ss.generate_scene_desc_dict()
+    for i in ss.scene_desc:
+        len(ss.scene_desc[i]) == 2
```
Comment (AlonDaks) [inline code/utils/tests/test_scene_slicer.py]: what does this line accomplish? The boolean expression is not being used

**Comment by AlonDaks** [comment]
Please make sure all files are yapf formatted `yapf -i <filename>.py` this will fix the >80 character lines

**Comment by AlonDaks** [comment]
Travis CI is failing bc you are referencing files that are not in the repo. You should create data files specific for testing, and refactor functions so that when u need a file, it's passed in to just the function where its being used that way you can modularly test it

File: code/utils/tests/test_scene_slicer.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+IS_DAY = 0
+IS_INT = 1
+
+
+def test_prepare():
+    data = np.array([[[[7, 9], [7, 8]], [[1, 2], [1, 8]]], [[[2, 3], [2, 1]],
+                                                            [[5, 4], [4, 3]]]])
+    img = nib.Nifti1Image(data, affine=np.diag([1, 1, 1, 1]))
+    nib.save(img, 'test_data.nii')
+    with open('scene.csv', 'w') as csvfile:
```
Comment (lisaannyu) [inline code/utils/tests/test_scene_slicer.py]: should be scenes.csv, same on line 23

**Pull request merged**